RSDKv5/main.cpp

```diff
@@ -1,17 +1,13 @@
-#include "RSDK/Core/RetroEngine.hpp"
-#include "main.hpp"
-
 #if RETRO_PLATFORM == RETRO_KALLISTIOS
-#if RSDK_DEBUG
-#include <arch/gdb.h>
-#endif
-
 #include <kos.h>
 
 KOS_INIT_FLAGS(INIT_IRQ | INIT_CONTROLLER);
 
 #endif
 
+#include "RSDK/Core/RetroEngine.hpp"
```

Comment:
Avoid moving these includes needlessly to make future upstream rebases a bit easier

Reply:
DUDE. There is something WRONG with their headers. If I put the kos include AFTER this include, it adds everything to namespace RSDK, and I can't call the KOS API properly. I have no idea wtf was happening, but I started to get pissy and didn't want to troubleshoot further, so I just stuck `<kos.h>` first... lol.

I guess I can try to figure it out a big harder... glad to hear you want to upstream. ;)

Reply:
I meant rebasing this fork with upstream, not upstreaming the port (would be a nightmare with the amount of ports available!).
The whole upstream code base will be updated to more closely match the original code thanks to the debug symbols in the Netflix port (we've got original namespaces, classes and a whole lot more), I just don't want SF94 to struggle even more trying to keep up with upstream!

Reply:
Alrighty, sounds good. You're the dude who's made this shit happen, and you've been doing a fine job at it and clearly know WTF you're doing so...

Whatever you say, bossman.

Reply:
I'm just a contributor turned maintainer, the real OGs who did the decompilation are Rubberduckycooly and stxtic ;)

Reply:
Badasses, all of them... Not gonna lie, I left game development and app-level development years ago to do back-end low-level SDK, driver, and OS shit.... but you guys are making me want to come out of retirement temporarily. ;)